Cargo.toml

```diff
@@ -15,3 +15,4 @@ cubeb-backend = "0.5"
 pulse-ffi = { path = "pulse-ffi" }
 pulse = { path = "pulse-rs" }
 semver = "^0.6"
+ringbuf = "0.1"
```

Comment:
License is okay (MIT/Apache2).  Uses SeqCst everywhere, so performance is probably not ideal.

Reply:
Yes, I'll get use something better based off the cubeb/gecko one (that I have optimized a bit locally to avoid false sharing), this is a bit crude. It should be straightforward to port it to rust, or to fix `ringbuf` to do the same.

We'll do something similar in some cases for the AudioUnit backend, when we can't use aggregate devices because the input and output stream are not on the same physical device.